### PR TITLE
API docs for Polymer 2.0

### PR DESCRIPTION
**Pull request by justinfagnani**
WIP (somewhat still). DO NOT MERGE

### CODE REVIEW

**Comment by arthurevans** [comment]
@justinfagnani Looking at the docs, one thing missing is instance properties, e.g., from Polymer.Element.

**Comment by arthurevans** [comment]
Also: almost all namespaces—Async, RenderStatus, Templatize, and so on—don't show any docs.

**Comment by arthurevans** [comment]
Pushed this to https://2-0-api-dot-polymer-project.appspot.com so I can reference URLs.

**Comment by justinfagnani** [comment]
@arthurevans I have a bunch of changes queued up to iron-doc-viewer. I'll ping when I push them.

Can you tell me how to push to GAE?

**Comment by notwaldorf** [comment]
I think we should merge this PR!

**Comment by notwaldorf** [comment]
@justinfagnani is there anything else you want to do to it?

File: gulpfile.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ gulp.task('minify-bundles', 'Minify element bundles',  ['build-bundles'], functi
     .pipe($.crisper({scriptInHead: false})) // split inline JS & CSS out into individual .js & .css files
     .pipe($.if('*.html', minifyHtml())) // Minify html output
     .pipe($.if('*.html', cssslam.gulp())) // Minify css in HTML output
-    .pipe($.if('*.js', uglifyJS())) // Minify js output
+    // .pipe($.if('*.js', uglifyJS())) // Minify js output
```
Comment (notwaldorf) [inline gulpfile.js]: Oh ugh, can we only skip the `iron-doc-viewer`, maybe? I think the rest of the files we want minified :(

File: package.json [hunk 2]
```diff
@@ -54,5 +55,8 @@
     "run-sequence": "^1.1.4",
     "toc": "PolymerLabs/node-toc",
     "yargs": "^3.29.0"
+  },
+  "dependencies": {
+    "clone": "^2.1.0"
```
Comment (notwaldorf) [inline package.json]: Why is this needed as a dep and dev dep?

**Comment by justinfagnani** [comment]
@monica can you deploy this to staging again?

File: package.json (hunk 2 above)
Comment (notwaldorf) [inline package.json]: @justinfagnani pls fix this after we merge?
